app/qml/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(MM_QML
     components/Symbol.qml
     components/TextWithIcon.qml
     dialogs/MigrateToMerginDialog.qml
-    dialogs/MissingAuthDialog.qml
+    dialogs/MMMissingAuthDialog.qml
```

Comment:
I think you. need to use it in-app somewhere

Reply:
Updated MissingAuthDialog to MMMissingAuthDialog in main.qml usage

Reply:
Ah you are right!